doc/rrdgraph_rpn.pod

```diff
@@ -204,7 +204,7 @@ I<percent>,I<count>,B<PERCENT>
 
 pop two elements (I<count>,I<percent>) from the stack. Now pop I<count> element, order them by size
 (while the smallest elements are -INF, the largest are INF and NaN is larger than -INF but smaller
-than anything else. No pick the element from the ordered list where I<percent> of the elements
+than anything else. Pick the element from the ordered list where I<percent> of the elements
 are equal then the one picked. Push the result back on to the stack.
```

Comment:
The English of this part of the sentence should probably be improved too: "are equal then the one picked."